packages/utxo-lib/tests/__fixtures__/compose.ts

```diff
@@ -524,6 +524,59 @@ export const composeTxFixture: Fixture[] = [
             type: 'final',
         },
     },
+    {
+        description:
+            'sorts the inputs randomly and puts change at random place between user-defined outputs' +
+            'when sortingStrategy=random',
+        request: {
+            changeAddress: { address: '1CrwjoKxvdbAnPcGzYjpvZ4no4S71neKXT' },
+            dustThreshold: 546,
+            feeRate: '10',
+            sortingStrategy: 'random',
+            outputs: [
+                {
+                    address: '1BitcoinEaterAddressDontSendf59kuE',
+                    amount: '150000',
+                    type: 'payment',
+                },
+            ],
+            utxos: [
+                {
+                    ...UTXO,
+                    vout: 2,
+                },
+                {
+                    ...UTXO,
+                    vout: 1,
+                },
+            ],
+        },
+        result: {
+            bytes: 374,
+            fee: '3740',
+            feePerByte: '10',
+            max: undefined,
+            totalSpent: '153740',
+            inputs: [
+                { ...UTXO, vout: 2 },
+                { ...UTXO, vout: 1 },
+            ],
+            outputs: [
+                {
+                    address: '1BitcoinEaterAddressDontSendf59kuE',
+                    amount: '150000',
+                    type: 'payment',
+                },
+                {
+                    address: '1CrwjoKxvdbAnPcGzYjpvZ4no4S71neKXT',
+                    amount: '50262',
+                    type: 'change',
+                },
+            ],
+            outputsPermutation: [0, 1],
```

Comment:
:thinking: This test case technically does cover the `'random'`, but if I'm not mistaken, this is the same result that would be obtained from `'none'`.
- it only tests that `randomSortingStrategy` works at all for the simplest case 
- but you could test a multi-output transaction, so there's really something to shuffle
  - I assume that change would have default index 2, but the mocked random would produce `[0, 2, 1]`, is that right? That would be interesting test case :slightly_smiling_face: